Sources/TuistGraph/Models/ProjectOptions.swift

```diff
@@ -18,18 +21,25 @@ extension Project {
         /// Text settings to override user ones for current project
         public let textSettings: TextSettings
 
+        /// Configures the name of the generated .xcodeproj.
+        public let xcodeProjectName: String?
+
```

Comment:
Where does this came from?

Reply:
Same reason as above. Added for consistency so that when types are converted back to `ProjectDescription` framework types, we don't loose information